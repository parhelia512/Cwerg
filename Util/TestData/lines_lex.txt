" ä ß "
"\"\""
"\n\n\"\"\\"
"\0\00\122\3"
"\x00\x01\x08\x10\x09\x02\x03\n\x11\x18 \x19\x12\x0b\x04\x05\x0c\x13\x1a!(0)\"\x1b\x14\x0d\x06\x07\x0e\x15\x1c#*1892+$\x1d\x16\x0f\x17\x1e%,3:;4-&\x1f'.5<=6/7>?"
"\x00\x00\x80?"
a b c d 11
[]
"a b" c "d 11"
a b [c d] 11
"a b" [c "d 11"]
"a b" [] c "d 11"
"a b" [a [b]]
"  # "
" # " # "  # "
# "aaa"
# "a
#,;;
"aaaaa" # sss
a , b
"jjjj
[aa
][
