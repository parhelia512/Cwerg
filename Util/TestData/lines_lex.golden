
" ä ß "
" \xc3\xa4 \xc3\x9f "

"\"\""
"\"\""

"\n\n\"\"\\"
"\n\n\"\"\\"

"\0\00\122\3"
"\x00\x00R\x03"

"\x00\x01\x08\x10\x09\x02\x03\n\x11\x18 \x19\x12\x0b\x04\x05\x0c\x13\x1a!(0)\"\x1b\x14\x0d\x06\x07\x0e\x15\x1c#*1892+$\x1d\x16\x0f\x17\x1e%,3:;4-&\x1f'.5<=6/7>?"
"\x00\x01\x08\x10\x09\x02\x03\n\x11\x18 \x19\x12\x0b\x04\x05\x0c\x13\x1a!(0)\"\x1b\x14\x0d\x06\x07\x0e\x15\x1c#*1892+$\x1d\x16\x0f\x17\x1e%,3:;4-&\x1f'.5<=6/7>?"

"\x00\x00\x80?"
"\x00\x00\x80?"

a b c d 11
{a} {b} {c} {d} {11}

[]
{[} {]}

"a b" c "d 11"
"a b" {c} "d 11"

a b [c d] 11
{a} {b} {[} {c} {d} {]} {11}

"a b" [c "d 11"]
"a b" {[} {c} "d 11" {]}

"a b" [] c "d 11"
"a b" {[} {]} {c} "d 11"

"a b" [a [b]]
@FAILED@


"  # "
"  # "

" # " # "  # "
" # " {# "  # "}

# "aaa"
{# "aaa"}

# "a
{# "a}

#,;;
{#,;;}

"aaaaa" # sss
"aaaaa" {# sss}

a , b
@FAILED@


"jjjj
@FAILED@


[aa
@FAILED@


][
@FAILED@

